server/src/main/java/io/envoyproxy/controlplane/server/LatestDiscoveryResponse.java

```diff
@@ -0,0 +1,49 @@
+package io.envoyproxy.controlplane.server;
+
+import java.util.Objects;
+import java.util.Set;
+
+class LatestDiscoveryResponse {
```

Comment:
This project uses [Google's AutoValues](https://github.com/google/auto/tree/master/value) for this type of objects. For consistency's sake I'd go for reusing this approach unless we want to unify and get rid of the magic related to autovalues from the project.

Reply:
Done